templates/tables/table.html.twig

```diff
@@ -0,0 +1,104 @@
+{#
+/**
+ * @file
+ * Theme override to display a table.
```

Comment:
What we do is removing the whole comment and just add a reference to the original template, like [this templates](https://github.com/openeuropa/oe_theme/blob/master/templates/content/node.html.twig) for an example.